### ISSUE TITLE
Bank Account - Arrow next to bank account selector is not clickable

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#35083]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.31-3
**Reproducible in staging?:** Y
**Reproducible in production?:**No, unable to check prod
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

**Issue found when executing PR** https://github.com/Expensify/App/pull/35032
## Action Performed:
1. Log in to New Expensify
2. Click on profile icon in LHN
3. Navigate to wallet > Add Bank account > Personal bank account
4. Select at least 2 accounts in Plaid flow, and finish the flow
5. Click on arrow pointing downwards next to Account field

## Expected Result:
Account dropdown with the list of available bank accounts should open

## Actual Result:
Clicking on arrow does not open or close the dropdown

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/d29b4ec1-a275-4b4b-a296-c5088386793d

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by dragnoir** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.

Arrow not pressable

### What is the root cause of that problem?
The issue is within the `<Picker>` component, the use of the native component `<RNPickerSelect>` where an issue already exist as mentioned here https://github.com/lawnstarter/react-native-picker-select/issues/237

### What changes do you think we should make in order to solve the problem?

Change the `<Picker>` component with a new page. When user click on the dropdown, a new page with the list of options will be opened. Like we do for the other options:

https://github.com/lawnstarter/react-native-picker-select/assets/12425932/d5cc3399-334d-4eb4-b000-4f3b7f7a5b3a

```js
<MenuItemWithTopDescription
                    title={------}
                    description={props.translate('-------')}
                    shouldShowRightIcon
                    disabled={-------}
                    onPress={() => Navigation.navigate(ROUTES.-------)}
                />
```

**Comment by dragnoir** [comment]
Then we need to fix it, it's a bad design on web currently

![image](https://github.com/Expensify/App/assets/12425932/3e362e65-8154-42f6-976d-fad8502bbefe)

**Comment by dragnoir** [comment]
A solution that fix the issue and make the UI better is preferable :)

**Comment by dragnoir** [comment]
@Krishna2323 how can you 

> adding zIndex:-1 to the icon

if the icon is from a native module where you don't have access to edit it!!!

**Comment by dragnoir** [comment]
@Krishna2323 from the doc of the component styling https://github.com/lawnstarter/react-native-picker-select#styling
You can't apply the style you mentioned on all the platforms. Your solution will not work on all devices.

**Comment by dragnoir** [comment]
Yes your ptoposal does not work on Android even with the workaround you mentioned.

The QA lmentioned above that this issue is within Android: Native also.

**Comment by dragnoir** [comment]
@aimane-chnaif do you want a solution to fix <Picker>?

**Comment by dragnoir** [comment]
@aimane-chnaif pls check https://github.com/Expensify/App/pull/34849

The team and the contributors are on the path to replace the `<Picker>` with the `<MenuItemWithTopDescription>` due to the bugs and issues related to `<Picker>`

On of the update is here:
https://github.com/Expensify/App/commit/c4cdb9b8d23283f4a3ea934a2117529856eec3c0#diff-18437544a71e140db50db4634f86560cda6a7f0af85c5d1ae4b4d9a2da1c0334L155

![image](https://github.com/Expensify/App/assets/12425932/f358a6bf-4a48-4223-b731-3708b32da833)

![image](https://github.com/Expensify/App/assets/12425932/32f8a01d-1534-4792-bda2-df04914cbb31)

This was about the Unit component and when I [tested this issue](https://github.com/Expensify/App/issues/35083#issuecomment-1911668654) the app was using the `Picker`.

I think it's better to use a better component and make the UI consistent better than having the Pciker with all the bugs and seeking a workaround to edit it. What do you think?

You mentioned we already use the Picker in oither places! I think we are going on a path to replace all of those with `<MenuItemWithTopDescription>`

**Comment by dragnoir** [comment]
@aimane-chnaif this one ?

![image](https://github.com/Expensify/App/assets/12425932/18563cc3-7903-48dc-a9f4-93a8331e896b)

Ther'is no issue there with the arrow icon, it's clickable without issue.

**Comment by dragnoir** [comment]
@aimane-chnaif all Picker are now working perfectly. Seems something with the last cleans on the patches.

Uploading 20240213_121221.mp4…

I think we can close this issue if it's just relevant to the click issue. But it's better to use the new component `<MenuItemWithTopDescription>` to make the app consistent between all screens and the login page language dropdown is working perfectly.

**Comment by Julesssss** [comment]
Thanks all

> I think we can close this as fixed unless we're going to refactor to push-to-page

That's going to need to be a separate discussion I think
